modules/core/server/views/email-templates/reference-notification-second.server.view.html

```diff
@@ -0,0 +1,7 @@
+{% extends './email.server.view.html' %}
+
+{% block content %}
+<a href="{{seeReferencesUrl}}">see my references</a>
+the recommendation was {{recommend}} <!-- yes, no, unknown -->
```

Comment:
This email template needs a design.

Reply:
I'd say it needs a copy; email templates already have a design by default:

<img width="673" alt="image" src="https://user-images.githubusercontent.com/87168/47421482-5a489e00-d789-11e8-9ab9-aa33142060a7.png">

You can see how the button is used from other email templates.

Reply:
Thank you. :+1: I don't know what those terms mean, therefore the confusion.

Reply:
Call for copy in https://github.com/Trustroots/trustroots/issues/718#issuecomment-431699368

/cc @Regenwasser @aoifehegarty